docs/containers.conf.5.md

```diff
@@ -219,6 +219,10 @@ Options are:
 Maximum number of processes allowed in a container. 0 indicates that no limit
 is imposed.
 
+**prepare_on_create**=false
+
+Prepare the container on creation for volume copy-up.
```

Comment:
How about something like:
```
Copy up the content from underlying image when a container with builtin volumes is created into the newly created volume. When false, the content will not be copied up until the container is started.
```

Reply:
Once this is agreed upon, then copy it into all three places.

Reply:
@mheon @TomSweeneyRedHat PTAL

Reply:
I will wait for @mheon or @TomSweeneyRedHat green signal before I update the description everywhere.